src/pages/Team.js

```diff
@@ -1,10 +1,40 @@
-function NotFound() {
+import { useState, useEffect } from "react";
+import { Box } from "@mui/material";
+import { Card } from "../components/team/Card";
+import { data } from "../components/team/mockData";
+
+function Team() {
+  // console.log(data);
+  const [members, setMembers] = useState(null);
+  useEffect(() => {
+    // getGithubData()
+    //   .then((data) => setMembers(data))
+    //   .catch((err) => console.error(err));
+    setMembers(data);
+  }, []);
+
+  // const renderArr = members;
+  console.log(members);
```

Comment:
Może warto wywalić tego console.log ?

Reply:
Zdecydowanie, ale jeszcze dużo poprawek będzie, tak że później to zrobię ;)